### DISCUSSION TITLE
Collections / playlists

### DISCUSSION THREAD

**Discussion by tooonyo** [OnionUI/Onion#501] [Ideas]
Hi ! 
I have a lot of roms and I want to sort them by genre (no matter which console it runs on). 
Do you think it's possible to make a sort of collection/playlist feature like in Steam for example ?
Instead of just having one list of favorites, you can create as much as you want and rename them. 
It would be great !
Thanks !

**Comment by Olywa** [comment]
It's not an update that would be quick or simple for the Onion team to achieve and would mean hacking/rewriting Miyoo binaries I think but there are a couple of workarounds:

1) The Roms/[systemanme]/ folders will work with a single level (only) sub folder hierarchy so taking GBA as an example, you could have:

**Roms/GBA/ShootemUps/** _(your Shoot 'em Ups here)_
**Roms/GBA/Platformers/** _(your Platformers here)_
**Roms/GBA/RPGs/** _(your GBA RPGs here)_
**Roms/GBA/** _(any other GBA roms here)_

2) You can use Retroarch with playlists (outside of the Onion UI) by using the Retroarch Standalone from the APP section.

3) OnionOS is very much dependant on the Miyoo binaries and if this was a feature their own stock firmware/OS in a future update then Onion would be able to make use of it. You can try to contact miyoo at [Email@lemiyoo.cn](mailto:Email@lemiyoo.cn) or by alixpress chat to ask for it to be added as a feature.

**Reply by tooonyo** [reply]
That's not exactly what I wanted because a "racing" folder can contain games from different systems.
I know somebody who developed a system that swap the "favorites" on the miyoo home screen so that you can create collections. It's a neat feature and very promising. With a little help form the community, it can become a feature of Onion ;-)

**Reply by Olywa** [reply]
@tooonyo there is indeed a feature being worked on by a user in the community that will swap favourites to provide multiple lists and he has some feedback and guidance from the Onion developers. We are watching to see how that develops.
